packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/grid.md

```diff
@@ -2,4 +2,26 @@
 id: Grid
 section: layouts
 ---
-Use a **Grid** to layout content on a fixed-size 12 column grid.
+The **grid** layout places content on a fixed 12 column grid.
+
+## Elements
+
+![Example of grid layout.](./img/grid-example.png)
+1. **Grid**: The grid container, which holds all grid items.
+1. **Grid item**: An individual section of content to be placed in the grid.
+
+## Usage
+
+Grid items can be ordered in a number of ways to create groups. The span value of all grid items must equal 12, but this can be achieved by combining different section sizes.
```

Comment:
```suggestion
Grid items can be ordered in a number of ways to create groups. The span value of all grid items must equal 12, but this can be achieved by modifying the span of each item to create different combinations of section sizes:
```